VideoWeb/VideoWeb/ClientApp/src/app/app.component.ts

```diff
@@ -94,13 +96,23 @@ export class AppComponent implements OnInit, OnDestroy {
             .getClientSettings()
             .pipe(first())
             .subscribe({
-                next: () => {
+                next: clientSettings => {
                     this.currentIdp = this.securityServiceProviderService.currentIdp;
+                    this.dynatraceService.addDynatraceScript(clientSettings.dynatrace_rum_link);
                     this.securityService.checkAuth(undefined, this.currentIdp).subscribe(async ({ isAuthenticated, userData }) => {
                         await this.postAuthSetup(isAuthenticated, false);
 
                         if (isAuthenticated) {
                             this.eventhubService.configureConnection();
+
+                            /* The line
+                            `this.dynatraceService.addUserIdentifyScript(userData?.preferred_username?.toLowerCase());`
+                            is calling a method `addUserIdentifyScript` from the `dynatraceService`
+                            service. This method is used to identify the user in Dynatrace
+                            monitoring by passing the user's preferred username in lowercase as a
+                            parameter.*/
+                            // TODO: uncomment the below line to add this identification
+                            // this.dynatraceService.addUserIdentifyScript(userData?.preferred_username?.toLowerCase());
```

Comment:
Can we not test this yet?

Reply:
I can't understand. Sorry

Reply:
If this is dependant on something else, can you reference the ticket number in the commented out code.

Reply:
done